opennms-container/minion/minion-config-schema.yml

```diff
@@ -201,58 +201,65 @@ categories:
             type:
               name: "string"
         label: "JVM Options"
-        description: "TODO"
+        description: "Additional Java options"
         key: "/process-env/java-opts"
-        example: "TODO"
+        example: "-Xrunjdwp:transport=dt_socket,server=y,suspend=n,address=7896"
   # IPC Configuration
   - name: "ipc"
     label: "IPC"
-    description: "TODO"
+    description: "Core, Minion and Sentinel distributed inter process communication (IPC) settings"
     categories:
       # IPC Configuration :: AWS
       - name: "aws"
         label: "AWS"
-        description: "TODO"
+        description: "Amazon Web Services Simple Queue Service (AWS SQS) configuration"
         config:
           - name: "properties"
             type:
               name: "properties"
             label: "Properties"
-            description: "TODO"
+            description: "AWS SQS properties"
             key: "/aws"
-            example: "TODO"
+            example: "aws_region: us-east-1, aws_access_key_id: xxxxxx or aws_secret_access_key: yyyyyy"
         categories:
           # IPC Configuration :: AWS :: SQS
           - name: "sqs"
             label: "SQS"
-            description: "TODO"
+            description: "IPC messaging configuration for AWS SQS"
             config:
               - name: "properties"
                 type:
                   name: "properties"
                 label: "Properties"
-                description: "TODO"
+                description: "AWS SQS IPC properties"
                 key: "/ipc/sqs"
-                example: "TODO"
+                example: "sink.MaximumMessageSize: 262144"
       # IPC Configuration :: gRPC
       - name: "grpc"
         label: "gRPC"
-        description: "TODO"
+        description: "gRPC for IPC configuration"
         config:
           - name: "host"
             type:
-              name: "host-with-port"
+              name: "host"
             label: "Host"
-            description: "TODO"
+            description: "gRPC host endpoint, typcally the same Core instance"
             key: "/ipc/grpc/host"
-            example: "TODO"
+            example: "my-opennms-instance-ip"
```

Comment:
@mattixtech the original was host-with-port. In our [docs](https://docs.opennms.com/horizon/28.0.0-SNAPSHOT/deployment/minion/install.html) I've found it with host and port separate. That's the reason I've added the additional config port section here. Does this make sense or should I revert it?

![Screenshot 2021-01-21 at 10 48 48](https://user-images.githubusercontent.com/1095181/105333647-409b9880-5bd6-11eb-9ee3-ec6149635d82.png)

Reply:
Yep, that looks better. We drop each of these values into the same cfg file written to by the example you showed so if the example is the right way to do it, your updates here reflect that.